documentation/design-docs/dotnet-tools.md

```diff
@@ -396,6 +396,85 @@ CONVERT
       Writing:       ./trace.speedscope.json
       Conversion complete
 
+
+### dotnet-stack
+
+SYNOPSIS
+
+    dotnet-stack [options] [command] [<args>]
+
+OPTIONS
+
+    --version
+        Display the version of the dotnet-trace utility.
+
+    -h, --help
+        Show command line help
+
+COMMANDS
+
+    report         Displays stack traces for the target process
+
+REPORT
+
+    dotnet-stack report -p|--process-id <pid>
+                        [-t|--report-type <report-type>]
+                        [-h|--help]
+
+    Prints the managed stack from every thread in the target process
+
+    -t, ---report-type <report-type>
+        The way to represent the stack data.  (default: standard)
+
+    -h, --help
+        Show command line help
+
+    Examples:
+      > dotnet-trace stack -p 1234
+      Stack for Thread (0x1):
```

Comment:
Are these IDs the OS id? The small numbers suggest that it isn't but I think it should be.

Reply:
I just put random numbers rather than convert the original numbers to their hex values.  I intend for these to be the OS thread Ids.